examples/main/app.js

```diff
@@ -150,7 +171,7 @@ class App extends PureComponent {
           {...mapViewState}
           onWebGLInitialized={ this._onWebGLInitialized }
           layers={this._renderExamples()}
-          effects={this._effects}
+          effects={effects ? this._effects : undefined}
```

Comment:
Can we use `null` instead of undefined to make it clear that we are returning a `null` object.